session-02/exercise.js

```diff
@@ -16,7 +16,13 @@ function transformArrayToUpper(listOfStrings) {
     the function should return 51
 */
 function sumOfAllAges(listOfStudentObjects) {
-  return;
+  let accumulatedAge = 0;
+  for(student of listOfStudentObjects){
```

Comment:
good code, but I recommend adding a `let` or `const` to this line `for(let student of listOfStudentObjects){`